frontend/app/templates/components/background-component/def-image-component.hbs

```diff
@@ -20,7 +20,7 @@
                     </div>
                 {{/each}}
             {{else}}
-                <div style="margin-left: 10px;" class="gallery">
+                <div style="margin-left: 10px; margin-bottom: 10px;" class="gallery">
```

Comment:
@rupav  Please don't use Inline CSS. As I see it was also present before your PR too. Can you please remove it and use class for the changes.